target/linux/rockchip/armv8/base-files/etc/board.d/02_network

```diff
@@ -56,6 +59,10 @@ rockchip_setup_macs()
 		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk*)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	sinovoip,rk3568-bpi-r2pro)
```

Comment:
no, this is duplicated. and this board has emmc module so we should generate macaddr from there.
```
	friendlyarm,nanopi-r2c-plus|\
	friendlyarm,nanopi-r4s|\
	friendlyarm,nanopi-r5s|\
	sinovoip,rk3568-bpi-r2pro)
```

Reply:
The board can boot from  eMMC  and SD card, so the correct is _mmcblk*_
```
friendlyarm,nanopi-r5c|\
sinovoip,rk3568-bpi-r2pro)
		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk*)
		lan_mac=$(macaddr_add "$wan_mac" 1)
```

Reply:
> The board can boot from eMMC and SD card, so the correct is _mmcblk*_

Current practice is to get a "fixed" mac address. The generated macaddr will be different if user replaces SD card, compared of that macaddr generated using eMMC module is more "stable".